apps/api/src/app/shared/helpers/liquid.ts

```diff
@@ -1,6 +1,13 @@
 import { Liquid } from 'liquidjs';
 
-export const parseLiquid = async (value: string, variables: object): Promise<string> => {
+export async function parseLiquid<T>(value: T, variables: object): Promise<T> {
+  const valueStringified = JSON.stringify(value);
+  const renderedString = await parseLiquidString(valueStringified, variables);
+
+  return JSON.parse(renderedString);
+}
+
+export async function parseLiquidString(value: string, variables: object): Promise<string> {
```

Comment:
A small question, is it possible some how to reuse the same liquid instance from the framework SDK? The reason I'm asking is because we might want to add custom liquid filters to the system, and in this case we will have to duplicate them twice.

Reply:
We can export it from `@novu/framework/internals`.